Validation/HGCalValidation/python/hgcalPlots.py

```diff
@@ -2467,7 +2467,7 @@ def _hgcalFolders(lastDirName="hgcalLayerClusters"):
 def append_hgcalTrackstersPlots(collection = 'ticlTrackstersMerge', name_collection = "TrackstersMerge"):
   # Appending generic plots for Tracksters
   hgcalTrackstersPlotter.append(collection, [
-              _hgcalFolders(collection+ "/" + hgcalValidator.label_TS._InputTag__moduleLabel)
+              _hgcalFolders(collection+ "/" + hgcalValidator.label_TS.configValue().replace("'",""))
```

Comment:
```suggestion
              _hgcalFolders(collection+ "/" + hgcalValidator.label_TS.value())
```
would suffice here, as in
https://github.com/cms-sw/cmssw/commit/6d94a740c51dd64265b006b6f7240e8e37a1405e#diff-a461420dd6f58cd4fb6d4571360a24c4c59ba8aaa6497c5e7e7b0dc74404a8b1

Reply:
@lecriste Thanks a lot Leo, you told me that but I forgot it.